packages/components/nodes/vectorstores/Milvus/Milvus.ts

```diff
@@ -136,10 +183,31 @@ class Milvus_VectorStores implements INode {
             const milvusUser = getCredentialParam('milvusUser', credentialData, nodeData)
             const milvusPassword = getCredentialParam('milvusPassword', credentialData, nodeData)
 
+            // tls
+            const secure = nodeData.inputs?.secure as boolean
+            const clientPemPath = nodeData.inputs?.clientPemPath as string
+            const clientKeyPath = nodeData.inputs?.clientKeyPath as string
+            const caPemPath = nodeData.inputs?.caPemPath as string
+            const serverName = nodeData.inputs?.serverName as string
+
+            // partition
+            const partitionName = nodeData.inputs?.milvusPartition ?? '_default'
+
             // init MilvusLibArgs
             const milVusArgs: MilvusLibArgs = {
                 url: address,
-                collectionName: collectionName
+                collectionName: collectionName,
+                partitionName: partitionName,
+                clientConfig: {
```

Comment:
question: if user dont have secure SSL and the pem path, does it still works?

Reply:
it doesnt work for me but I made it secure in Milvus configurations, maybe with different configurations on Milvus config (i.e tls_mode: 1 instead of 2) it might work. Anyhow for my case its crucial.